config/checkstyle/suppressions.xml

```diff
@@ -9,5 +9,5 @@
   <suppress checks="MissingJavadocMethodCheck" files=".*Test\.java"/>
   <suppress checks="CommentsIndentation" files=".*\.java"/>
   <suppress checks="DeclarationOrder" files=".*\.java"/>
-
+  <suppress checks="OperatorWrap" files=".*\.java"/>
```

Comment:
What does this do?

Reply:
There is a warning that the operator (like + for string concatenation) must be on a new line, but I haven't found a way to make IntelliJ auto-format for that, and it's really minor so I will just disable it for now

Reply:
Hmm I think we should enable it before merging right?

Reply:
Hmm because I haven't been able to get the IntelliJ to conform to this rule, and this is very minor (it's just the position of the operator) so I think it's best to disable it for now (if not forever)

Reply:
Ok, but I think before we submit v1.3 on Friday we should remove this